client/app/components/RatingsAndReviews/ReviewList/ReviewButtonsContainer/AddReviewButton/ReviewCharacteristics/ReviewCharacteristic.jsx

```diff
@@ -2,86 +2,134 @@
 import React, { useState } from 'react';
 
 const ReviewCharacteristic = ({ name, id, reviewCharsObj, setReviewCharsObj }) => {
-  const [sizeCharacteristic, setSizeCharacteristic] = useState(0);
-  const [widthCharacteristic, setWidthCharacteristic] = useState(0);
-  const [comfortCharacteristic, setComfortCharacteristic] = useState(0);
-  const [qualityCharacteristic, setQualityCharacteristic] = useState(0);
-  const [lengthCharacteristic, setLengthCharacteristic] = useState(0);
-  const [fitCharacteristic, setFitCharacteristic] = useState(0);
+  // const [sizeCharacteristic, setSizeCharacteristic] = useState(0);
+  // const [widthCharacteristic, setWidthCharacteristic] = useState(0);
+  // const [comfortCharacteristic, setComfortCharacteristic] = useState(0);
+  // const [qualityCharacteristic, setQualityCharacteristic] = useState(0);
+  // const [lengthCharacteristic, setLengthCharacteristic] = useState(0);
+  // const [fitCharacteristic, setFitCharacteristic] = useState(0);
```

Comment:
Make sure to delete your comments. This won't fail you a code review but if you're already going to check for the console log might as well fix this too